man/systemd.netdev.xml

```diff
@@ -1137,7 +1141,15 @@ Name=dummy-test
 Kind=dummy
 MACAddress=12:34:56:78:9a:bc</programlisting>
     </example>
+    <example>
+      <title>/etc/systemd/network/25-vrf.netdev</title>
+      <programlisting>[NetDev]
```

Comment:
Please add an explanation what the example does (see my comments on previous version).